extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -153,6 +153,9 @@ let sraRequests = null;
     }} */
 let TroubleshootData;
 
+/** @private {?JsonObject} */
+let windowLocationQueryParameters = null;
```

Comment:
nit: don't think you need to instantiate to null here, you could just do let windowLocationQueryParameters.

Reply:
Done